src/cell/build.c

```diff
@@ -433,6 +437,8 @@ build_init_dir(build_t* it)
 		visor_prompt(it->visor, "title of new game?", title, sizeof title);
 		visor_prompt(it->visor, "author's name?", author, sizeof author);
 		visor_prompt(it->visor, "one-line summary?", summary, sizeof summary);
+		visor_prompt(it->visor, "screen width (default is 320)?", width, sizeof width);
+		visor_prompt(it->visor, "screen height (default is 240)?", height, sizeof height);
```

Comment:
How hard would it be to merge these into a single prompt, asking for the res in `WxH` format? Only because I find it weird to allow a single component to default if the other is explicitly provided.

Reply:
Originally I was going to do that, I'm not sure why I changed it. And the reason it has a default is because it's the one (as far as I know at least) that's a hard requirement for the engine.

Reply:
Oh yeah, I don’t mind there being a default - I just found it weird to have a “split default”. I’m not sure why I even wrote the template that way, tbh. Thanks for changing it though